packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -11,6 +11,12 @@ export type FieldValue = {
     value: string | number | boolean | null;
 };
 
+export type WfoRadioDropdownOption<T> = {
+    label: string;
+    id: string;
+    value: T;
+};
+
```

Comment:
I would suggest to keep this in the component, it is not domain data or something that is super-reusable (think about the SortOrder enum)